__tests__/index.tests.js

```diff
@@ -39,3 +40,28 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              expect.arrayContaining(["GET /api", "GET /api/topics","GET /api/articles",  ]))
+              
+                     
+          });
+      });
+    }
+    )
+    describe('error 404 GET /api/topics/', ()=>{
```

Comment:
This 404 is not tied to this end point in any way, consider a better test description for an incorrect route.